cabot/templates/cabotapp/_base_form.html

```diff
@@ -24,10 +26,5 @@
   </div>
   {% else %}
```

Comment:
Is this still necessary?

Reply:
looks like that was an existing issue not created by this PR, so i think i'm just going to merge

Reply:
Fair enough